journey-map/public/index.html

```diff
@@ -34,6 +34,45 @@ <h1>
         </div>
       </div>
       <div id="header-content">
+        <form id="transitions-traffic-form">
+            <h2>Transitions Traffic</h2>
+            <fieldset id="journeyTransitionsInput">
+              <legend>Customise api</legend>
```

Comment:
```suggestion
              <legend>Define traffic</legend>
```
I think makes more sense as a header. Customise API sounds like it does something different